host/tests/L3_TestCases/dsAudio/dsAudio_test01_EnableDisableAndVerifyAudioPortStatus.py

```diff
@@ -25,37 +25,43 @@
 import sys
 
 dir_path = os.path.dirname(os.path.realpath(__file__))
-sys.path.append(dir_path+"/../")
+sys.path.append(os.path.join(dir_path, "../../"))
 
 from dsClasses.dsAudio import dsAudioClass
 from raft.framework.plugins.ut_raft import utHelperClass
 from raft.framework.plugins.ut_raft.configRead import ConfigRead
 from raft.framework.plugins.ut_raft.utPlayer import utPlayer
 from raft.framework.plugins.ut_raft.utUserResponse import utUserResponse
 
-class dsAudio_test1_EnableDisableAndVerifyAudioPortStatus(utHelperClass):
+class dsAudio_test01_EnableDisableAndVerifyAudioPortStatus(utHelperClass):
```

Comment:
upgrade to consolidate functions that are repeated into a helperClass specificl for this module.